.github/workflows/deploy_snapshot.yml

```diff
@@ -16,7 +25,4 @@ jobs:
         uses: actions/checkout@v2
       - name: Deploy new snapshot of DeepLinkDispatch
         shell: bash
-        env:
-          SONATYPE_NEXUS_USERNAME: ${{ secrets.NEXUS_USER }}
-          SONATYPE_NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
-        run: ./gradlew uploadArchives
+        run: ./gradlew publishAllPublicationsToMavenCentral
```

Comment:
if someone changes the version to a non snapshot version number, then this will publish that real version (since it has the ability to close and release the repository, which previously didn't exist)

Maybe we also need to verify the version ends in -SNAPSHOT before running this action?

docs https://vanniktech.github.io/gradle-maven-publish-plugin/central/#publishing-snapshots